include/oneapi/dpl/pstl/parallel_backend_serial.h

```diff
@@ -45,7 +45,7 @@ __cancel_execution(oneapi::dpl::__internal::__serial_backend_tag)
 template <class _ExecutionPolicy, class _Index, class _Fp>
 void
 __parallel_for(oneapi::dpl::__internal::__serial_backend_tag, _ExecutionPolicy&&, _Index __first, _Index __last,
-               _Fp __f)
+               _Fp __f, std::size_t /*__grainsize*/)
```

Comment:
I think we (interestingly) need a default argument for the unnamed parameter. (see the error(s) in the serial backend)

``` 
__parallel_for(oneapi::dpl::__internal::__serial_backend_tag, _ExecutionPolicy&&, _Index __first, _Index __last,
               _Fp __f, std::size_t /*__grainsize*/ = 1)
```

Apparently it is legal.

Reply:
good catch